lib/pure/random.nim

```diff
@@ -19,69 +19,63 @@
 ## Basic usage
 ## ===========
 ##
-## To get started, here are some examples:
-##
-## .. code-block::
-##
-##   import random
-##
-##   # Call randomize() once to initialize the default random number generator
-##   # If this is not called, the same results will occur every time these
-##   # examples are run
-##   randomize()
-##
-##   # Pick a number between 0 and 100
-##   let num = rand(100)
-##   echo num
-##
-##   # Roll a six-sided die
-##   let roll = rand(1..6)
-##   echo roll
-##
-##   # Pick a marble from a bag
-##   let marbles = ["red", "blue", "green", "yellow", "purple"]
-##   let pick = sample(marbles)
-##   echo pick
-##
-##   # Shuffle some cards
-##   var cards = ["Ace", "King", "Queen", "Jack", "Ten"]
-##   shuffle(cards)
-##   echo cards
-##
+runnableExamples:
+  # Call randomize() once to initialize the default random number generator.
+  # If this is not called, the same results will occur every time these
+  # examples are run.
+  randomize()
+
+  # Pick a number between 0 and 100.
```

Comment:
```suggestion
  # Pick a number in 0..100.
```
(english is not precise enough, 0..100 is better)

Reply:
How will someone that doesn't have a background in- well, hard to say what really- know what `..` means?

We should use English here, if it's not accurate then we can certainly make it so at the expense of verbosity.

Reply:
`..` vs `..<` is litterally used everywhere in nim, it's what you learn within the 1st hour.
A user who encounters `std/random` will have encountered `..` long before.
`between 0 and 100` immediately raises the question of whether 100 is inclusive or exclusive, whereas `0..100` is un-ambiguous.